client/src/components/pages/home/HomeBanner.css

```diff
@@ -0,0 +1,196 @@
+/* TODO: Might need some cleanups */
+
+.HomeBannerBackground-container {
+  position: absolute;
+  overflow: hidden;
+  background-color: #38414f;
+  width: 100%;
+  height: 100%;
+  z-index: 0;
+}
+
+.HomeBannerBackground-stripes-left {
+  position: absolute;
+  top: 0px;
+  transform: scaleY(-1);
+  opacity: 0.1;
+}
+
+.HomeBannerBackground-stripes-right {
+  position: absolute;
+  bottom: 0px;
+  right: 0px;
+  /* transform: scaleX(0.7) scaleY(-0.7) translate(30%, -30%); */
+  transform: scaleY(-1);
+}
+
+.HomeBannerBackground-striped-circle-white {
+  position: absolute;
+  top: 0px;
+  right: -64px;
+  scale: 0.5;
+}
+
+.HomeBannerBackground-striped-circle-orange {
+  position: absolute;
+  bottom: -96px;
+  left: -128px;
+  scale: 0.7;
+}
+
+.HomeBanner-container {
+  position: relative;
+  display: flex;
+  justify-content: stretch;
+  align-items: stretch;
+  margin-bottom: var(--l);
+  flex-direction: column;
+  padding-bottom: 48px;
+}
+
+.HomeBanner-container > * {
+  z-index: 1;
+}
+
+.HomeBanner-section-1 {
+  display: flex;
+  padding: 30px;
+}
+
+.HomeBanner-section-2 {
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  overflow: hidden;
+}
+
+.HomeBanner-logo {
+  width: 103px;
+  height: 103px;
+}
+
+.ant-typography.HomeBanner-title {
+  color: var(--gts-orange);
+  font-size: 30px;
+  font-weight: 900;
+  margin-left: 30px;
+  line-height: 25px;
+}
+
+.ant-typography.HomeBanner-description {
+  margin-top: 12px;
+  font-weight: 500px;
+  font-size: 14px;
```

Comment:
text could probably be bigger(?)
![image](https://github.com/cory2067/gtsosu/assets/8433005/dc594baa-7d22-4e23-9c97-1ab67315d28c)

Reply:
Increased slightly in 400d747fcda341fb788f49c155bcf464530e1af2
Currently there's only 1 stage of font size after the layout change, which makes the text look somewhat smaller in wider viewports
This is how it looks after:
![image](https://github.com/cory2067/gtsosu/assets/5846322/ad5d01ed-b2d1-475b-b228-0f00c3fb6e81)
![image](https://github.com/cory2067/gtsosu/assets/5846322/832a46ea-22cf-4180-a5c8-65e7f0812724)